src/uct/ib/dc/dc_mlx5_ep.h

```diff
@@ -27,35 +27,35 @@
 
 enum uct_dc_mlx5_ep_flags {
     /* DCI pool EP assigned to according to it's lag port */
-    UCT_DC_MLX5_EP_FLAG_POOL_INDEX_MASK     = UCS_MASK(3),
+    UCT_DC_MLX5_EP_FLAG_POOL_INDEX_MASK     = UCS_MASK(5),
```

Comment:
maybe define (or static assert) UCT_DC_MLX5_IFACE_MAX_DCI_POOLS according to UCT_DC_MLX5_EP_FLAG_POOL_INDEX_MASK?